packages/math/src/ObservablePoint.ts

```diff
@@ -9,13 +11,18 @@
  */
 export class ObservablePoint
 {
+    public cb: Function;
+    public scope: object;
+    protected _x: number;
+    protected _y: number;
+
     /**
      * @param {Function} cb - callback when changed
      * @param {object} scope - owner of callback
      * @param {number} [x=0] - position of the point on the x axis
      * @param {number} [y=0] - position of the point on the y axis
      */
-    constructor(cb, scope, x = 0, y = 0)
+    constructor(cb: Function, scope: object, x = 0, y = 0)
```

Comment:
`Function` should be a type describing the actual function prototype, like `() => void`.

`object` is the equivalent of `{}`. This should be `any` instead.

If you want to get fancy, this could instead be `constructor<T>(cb: (this: T) => void, scope: T, x = 0, y = 0)` so that Typescript expresses that the callback is called with `scope` as `this`.

Reply:
doing the fancy approach doesnt work on the constructor. So ive done `cb: ()=> any, scope: any`

Reply:
Sorry it has to be on the class not the ctor, I just put it there for comment brevity:

```ts
class ObservablePoint<T>
{
    public scope: T;
    constructor(cb: (this: T) => void, scope: T, x = 0, y = 0)
    {
    }
}
```